### PR TITLE
Harmonize all ndimage usage across the library

### PR DESCRIPTION
**Pull request by jni**
Only two forms remain in use:

- `from scipy import ndimage as ndi`
- `from scipy.ndimage import function`

### CODE REVIEW

**Comment by tacaswell** [comment]
That looks like no fun at all....

**Comment by jni** [comment]
@tacaswell what? I *love* doing repetitive but always slightly different regex searches and substitutions across a whole library... =P

**Comment by JDWarner** [comment]
For the record, I have nothing against avoiding an alias entirely in favor of `ndimage`, though at times the extra characters make holding to 80 chars per line awkward.

**Comment by vighneshbirodkar** [comment]
Could you also update the import conventions ? 
https://github.com/scikit-image/scikit-image/blob/master/CONTRIBUTING.txt#L144

**Comment by jni** [comment]
@JDWarner I do have something against avoiding aliases. =)

@vighneshbirodkar done, thanks for the catch!

File: skimage/transform/_geometric.py [hunk 1]
```diff
@@ -2,11 +2,10 @@
 import math
 import warnings
 import numpy as np
-from scipy import ndimage, spatial
+from scipy import ndimage as ndi, spatial
```
Comment (JDWarner) [inline skimage/transform/_geometric.py]: Let's break these into separate lines for `ndi` and `spatial`.

**Comment by JDWarner** [comment]
@jni I really thought this was more standardized! I have only the one comment.

**Comment by jni** [comment]
@JDWarner you and I both! I think this is part of the advantage of scikit-image's modularity. =) Fixed!

**Comment by JDWarner** [comment]
Thanks, @jni!

**Pull request merged**